content/guides/tree-shaking.md

```diff
@@ -84,3 +84,5 @@ function(e,t,n){"use strict";function r(e){return e*e*e}t.a=r}
 /* ... */
 function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(0);console.log(n.i(r.a)(5))}
 ```
+
+T> Note, using the `--optimize-minimize` flag is NOT required for tree shaking to occur. Tree shaking will occur when using a production configuration. This can also be configured directly in a `webpack.config.js` file. Read [production build](/guides/production) for an in-depth guide on how to create a production config file.
```

Comment:
Instead of saying:

> Tree shaking will occur when using a production configuration.

I think we should be more specific on exactly what's necessary which I think, as I mentioned in #1473, is allowing webpack to actually resolve `import` / `export` statements (i.e. disabling any transpilers like babel from resolving them). And, including the `UglifyJsPlugin`. The `import` / `export` is already mentioned at the top of this guide so maybe just noting that either `--optimize-minimize` should be used (which includes the `UglifyJsPlugin` under the hood) or the plugin can be configured directly (likely in a "production" configuration).